### DISCUSSION TITLE
请问怎么拿到field下的组件实例呢？

### DISCUSSION THREAD

**Discussion by cyt68** [alibaba/formily#3379] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/20410138/188869710-649eda60-5de9-42b3-92fc-b508141b902a.png)
请问怎么拿到field下的组件实例，比如图片所示，我想拿到组件Input的实例。【用途是为了调用组件上的方法】
通过ref或者别的方式能够实现吗？感谢

**Comment by yiyunwan** [comment]
`field.setComponentProps({
                  ref: markRaw({
                    current: null,
                  }),
                })`
